core/src/main/java/nl/nn/adapterframework/configuration/AdapterManager.java

```diff
@@ -145,6 +147,19 @@ public Adapter getAdapter(int i) {
 		return startedAdapters;
 	}
 
+	@Override
+	public void configure() {
+		log.info("configuring all adapters for AdapterManager "+this+"]");
```

Comment:
Ik zou in deze melding verwijzen naar de configuratie, dus: ``configuring all adapters for configuration [ ... ]``

Reply:
De adapterManager is niet direct bewust bij welke configuratie hij hoort. Moet dat? Lijkt mij een onnodige injectie puur voor een log.info

Reply:
Als je een message logt op INFO niveau, dan mag je toch verwachten dat hij informatief is.